docs/guides/authentication.md

```diff
@@ -48,3 +48,69 @@ class Mutation:
 
         return LoginSuccess(user=User(username=username))
 ```
+
+### Access authenticated user in resolver
+
+Its fairly common to require user information within a resolver. We can do that in a type safe way with a custom context dataclass.
+
+For example, in fastapi this might look like this:
+
+```python
+from functools import cached_property
+from typing import cast
+
+import strawberry
+from fastapi import FastAPI, Request
+from strawberry.fastapi import BaseContext, GraphQLRouter
+from strawberry.types import Info as _Info
+from strawberry.types.info import RootValueType
+
+
+class User:
+    pass
```

Comment:
Can you add a comment explaining that this is just a stub for an actual user object?